pages/admin/users.tsx

```diff
@@ -70,10 +70,15 @@ const AdminUsers: React.FC<QueryDataProps<AllUsersData>> = ({ queryData }) => {
           onClick={(value: string) => changeFilter(value, 'option')}
           currentOption={searchOption.option}
         >
-          Search By:
+          <label htmlFor="search">Search By:</label>
```

Comment:
What does this do, does it make the UI look better?

Reply:
UI looks the same but accessibility is better, you can select input by label and use better testing library query (and it's [recommended](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names) label inputs anyway).